integration_tests/blockstack_integration_tests/scenarios/name_pre_regup_subdomain_lookups.py

```diff
@@ -161,6 +162,18 @@ def scenario( wallets, **kw ):
 
     # query each subdomain
     proxy = testlib.make_proxy()
+
+    # test 301 redirects.
+    res = testlib.blockstack_REST_call('GET', '/v1/names/baz.foo1.test', ses)
+    if 'error' in res:
+        res['test'] = 'Failed to query non-registered name.'
+        print json.dumps(res)
+        return False
+    if res['http_status'] != 301:
+        res['test'] = 'Failed to get a redirect.'
+        print json.dumps(res)
+        return False
+
```

Comment:
For completeness, can you also have this test case do the following?
* spawn a subdomain registrar
* register a subdomain via the subdomain registrar (so it's in its queue)
* use `blockstack_REST_call()` to fetch it via the subdomain registrar (i.e. ensuring that the 301 correctly connects `requests` to the subdomain registrar and serves back the subdomain state)

Reply:
I can do this as well, but if so, please open an issue so I remember to do it :)

Reply:
Sure -- I can do this. Will do it in a different scenario though.

Reply:
Added as `blockstack_integration_tests.scenarios.subdomain_registrar_redirect_test`